test/functional/apps/discover/_discover.js

```diff
@@ -157,9 +159,9 @@ export default function ({ getService, getPageObjects }) {
         await PageObjects.visualize.waitForVisualization();
 
         const newDurationHours = await PageObjects.timePicker.getTimeDurationInHours();
-        if (newDurationHours < 1 || newDurationHours >= 5) {
-          throw new Error(`expected new duration of ${newDurationHours} hours to be between 1 and 5 hours`);
-        }
```

Comment:
dear @ryankeairns, just wanted to know if there's a reason for this check. from my point of view, when rounding the interval, it's always 3 hours. just wanted to be sure, that I don't mess up by changing this (it worked locally, let's see what Mr. Jenkins says, maybe that's the reason for this check)